paimon-core/src/main/java/org/apache/paimon/table/system/ManifestsTable.java

```diff
@@ -233,6 +232,10 @@ private StatsManifestsGetter(FileIO fileIO, Path location, Table dataTable) {
 
         private void initialize() {
```

Comment:
Just convert `StatsManifestsGetter` to a static method?

Reply:
I mean we can do a bit of refactoring, here the class doesn't seem to be of much use, just a static method will do for our needs.

Reply:
Sorry, I misunderstood, I have added a static method.